ops/terraform/services/base/values/prod-sbx.yaml

```diff
@@ -8,8 +8,7 @@
 /bfd/${env}/migrator/sensitive/db/username: <<CIPHER>>AgV4g_U94LJ34bsgSn2lEvOWBq7xoEBOVdrbB151l4SGP98AdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREFsdjFJS3NmYjJqL1c4K0JRQWxzamtKZWF1Rnh5a1ZLK3Q1bnBZM21odWMreG53ZTgxdFVkQ2dXYnRka1dteFFnUT09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay1jMjg5NTM3ODY2MzY0NTk4OTQ2NWQzMTJmYjcxMGU0NQC4AQIBAHjqKCGUsrfafVye_2mw4SSOkeKVkIdNH7fvxxo_a9cGpQEriVIUHXWFS5t6TjdW7oW7AAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMcQ7zduLGcRM69iwBAgEQgDsyI2hG8gIs7Y3ue0jGeGulvPyTZk_-DBfaEtbLtp8Av53AsoGGgEO77DN7FrxWNOBdKluFl-9kOyw_LAIAABAAwA5XNOSutYpQpRUcgyePJCYz4d_VGtI5qtXhjs2A3FnMN27shKJbO__yB0DaZYqq_____wAAAAEAAAAAAAAAAAAAAAEAAAAT4x9l8JX9s-rfYZd5wTN7H9lZYyCWgvQo9AhXL-hmbOJ3HSAAZzBlAjAx5S6YmaNp6uAoGXMKfKiKe8wwITU3R8ELm9urrCRwEaSE1Ba1SeZShoaG2n0GmjcCMQDWEnD0d2JvAdCrUI_kFApOmd_uXykETkQEVVi6L7HCK7kn90dhN3jt3NPMMm9Udm0=<</CIPHER>>
-/bfd/${env}/migrator/sensitive/db_migrator_db_password: <<CIPHER>>AgV4YZOldSwARk2jgb8NvDxzEoVBv99UeUd3Cv_LcPXxGasAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREE4VSsvU3BldnZ2bXZsVVZpYWY3ZC90K0pnUlUxaWRkTWxhcGYrbmZ5R0V4M1cxbDBhSWVOR1ZWeEsrTGhBVzJsUT09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay1jMjg5NTM3ODY2MzY0NTk4OTQ2NWQzMTJmYjcxMGU0NQC4AQIBAHjqKCGUsrfafVye_2mw4SSOkeKVkIdNH7fvxxo_a9cGpQFuThC7aKONHggv1pBuCTR0AAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMwZ28tJkA1xlpuTESAgEQgDu_Y13E4et51JvpIZGum5RUWU1uq5neQEIqbuE7WZLhmG0CUyBGntxa2Eb4RWJLZ7Bs54b9gnrW-Hl7LQIAABAAi7ciwAHKk4GBCPGmHob-ttbxXC8CrNsWLCnOV4UUSX0Gz9DTBPKwo034xMZToFgZ_____wAAAAEAAAAAAAAAAAAAAAEAAAAYL3CcyjuuyGuD94wmiXwnEMVA9jhiZ60jgsJPpbXZYsanRFQHV3-64gBnMGUCMBEJjbEKLTN0JkvgxQjMBzdBb1Us8Np878ZjX1KA0RaHJpuuaaLV4NC7GQVNZAMp2wIxAJNU4rju8U7MdqnVfBnZqJgL2lF1cRRTRvXZf5_Tc954woEFVghShAgFFQsKcB8oLw==<</CIPHER>>
-/bfd/${env}/migrator/sensitive/db_migrator_db_username: <<CIPHER>>AgV4vgdUgnDSkEDWE6mgfmHW_ln1JnXVbx4tnAhxZwPw15wAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREEzU2RJV2lMLytWM215TTdNZmxHUWUybHZ5eGRHYUt3MUlkR1prMXE4SlVvNUJFa0ZwMGRZVXc2b1U3TWtXR2hoZz09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay1jMjg5NTM3ODY2MzY0NTk4OTQ2NWQzMTJmYjcxMGU0NQC4AQIBAHjqKCGUsrfafVye_2mw4SSOkeKVkIdNH7fvxxo_a9cGpQETcXD4abz7JSrHmnc3Rt15AAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMSj5oYdTdqVMTCf6-AgEQgDtAizh7RVS7u6_t-4Xm5AT8HFIWm5bpynDnxAyxJD0kyDm_9iQnTQBdVnTHQhgrCL6Oys7QiwO-Hxeu9QIAABAANV2ZHdC4IG1ar6e1F2Wm_HIYkEq6AegkxmX1WR_Mcm9TDFrHByQd73HBnj6jo9i5_____wAAAAEAAAAAAAAAAAAAAAEAAAATR8HiIeY773hqrkXUvnNHrMLZBafYwxdt_NQdqBJi0s5B0dQAZzBlAjBGCyTqE9uXTVoGsmIM-wZ5vRa_ajybyleIEXnkdZFjXF0j32_JC4aOySHXJcDK55YCMQCTUK8ciY9xNBVyeEqtfF9UO2Je5v4BNH0mRheobCrEzVeiMBnhI4Gxus4NoBszAWk=<</CIPHER>>
```

Comment:
I think we may need to update `ephemeral.tf` to account for some of these removals.

Reply:
> I think we may need to update `ephemeral.tf` to account for some of these removals.

Thanks! Updated.